js/hw_3_js_react_1/src/utils.js

```diff
@@ -0,0 +1,100 @@
+export const getClassName = (rowId,cellId,snake,food,extraFood) => {
+    let className = 'cell';
+    for(let i = 0; i < snake.length;i++){
+      if(snake[i][0] === rowId && snake[i][1] === cellId){
+        className = 'cell snake';
+      }
+    }
+    
+    if(food[0] === rowId && food[1] === cellId){
+      className = 'cell food';
+    }
+     
+    
+    if(extraFood !== undefined && extraFood[0] === rowId && extraFood[1] === cellId){
+      className  = 'cell bonus-food';
+    }
+  
+  
+    return className;
+  };
+
+
+  export function generateFood(snakeBody,food) {
```

Comment:
It is possible to have a single function to handle both extra-food and normal food. Think about it.